lua/catppuccin/groups/integrations/feline.lua

```diff
@@ -71,7 +71,11 @@ function M.setup(opts)
 		opts.sett = opts.sett or {}
 		opts.mode_colors = opts.mode_colors or {}
 	else
-		opts = {}
+		opts = {
+			assets = {},
+			sett = {},
```

Comment:
I think this should be `set` ?

Reply:
In the current implementation:
if you use the documentation to create the feline config:
```
local ctp_feline = require('catppuccin.groups.integrations.feline')

ctp_feline.setup()
```

What will happen is: 

- a new empty table will be created on line 74: ```opts = {}```.
- after that on line 80: ```assets = vim.tbl_deep_extend("force", assets, opts.assets)``` this error will be thrown: "vim/shared.lua:0: after the second argument: expected table, got nil" because the opt.assets is nil. 

This change eliminates this issue;

Note: 

- this does not happen if you pass an empty table to ctp_feline so updating the documentation would have the same effect.

Reply:
Sorry, didn't realise the variable is meant to be `sett`

Thanks for the info